packages/editor/src/components/post-schedule/style.scss

```diff
@@ -9,11 +9,12 @@
 	}
 }
 
-.editor-post-schedule__dialog-toggle {
+.editor-post-schedule__dialog-toggle.components-button {
 	display: block;
 	max-width: 100%;
 	overflow: hidden;
 	text-align: left;
-	text-overflow: ellipsis;
-	white-space: nowrap;
+	white-space: unset;
+	height: auto;
+	padding: 10px 12px;
```

Comment:
This padding is creating a 35px button for me. Maybe use `min-height: $button-size` instead?

Reply:
min height doesn't work properly for multi-line buttons. We can have both though.

Reply:
Ok so I added a line height using pixels and a comment explaining why.

Reply:
I see what you mean. Seems fine now:

<img width="399" alt="Screenshot 2023-11-20 at 16 49 10" src="https://github.com/WordPress/gutenberg/assets/846565/077739ca-a245-4725-95ef-e1eaa4455705">

Should we us the `$grid-unit-*` vars?

Reply:
Trying calculations.